### PR TITLE
Kafka develop v3

### PR DESCRIPTION
**Pull request by sowmya695**

### CODE REVIEW

File: registration-processor-default.properties [hunk 1]
```diff
@@ -984,3 +984,5 @@ mosip.registration.processor.lostrid.max-registration-date-filter-interval=30
 
 ##Ability to update contact information as part of LOST UIN
 uingenerator.lost.packet.allowed.update.fields=phone,email,permanentAddress
+logging.level.io.mosip.registration.processor.core.eventbus.KafkaMosipEventBus=DEBUG
+registration.processor.healthcheck.timeout=5000
```
Comment (vishwa-vyom) [inline registration-processor-default.properties]: Lets rename this property to mosip.regproc.health-check.handler-timeout

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: registration-processor-default.properties [hunk 2]
```diff
@@ -984,3 +984,5 @@ mosip.registration.processor.lostrid.max-registration-date-filter-interval=30
 
 ##Ability to update contact information as part of LOST UIN
 uingenerator.lost.packet.allowed.update.fields=phone,email,permanentAddress
+logging.level.io.mosip.registration.processor.core.eventbus.KafkaMosipEventBus=DEBUG
+mosip.regproc.health-check.handler-timeout=5000
```
Comment (vishwa-vyom) [inline registration-processor-default.properties:990]: can you add comment on the above configs, 
for the first one lets say this is temporary added for debuging.
for the second one lets explain what is the config about and what is the unit of timeout

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: registration-processor-default.properties (hunk 1 above)
Comment (sowmya695) [reply registration-processor-default.properties]: fixed

File: registration-processor-default.properties (hunk 2 above)
Comment (sowmya695) [reply registration-processor-default.properties:990]: fixed

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**